tests/pytests/test_data_naif.py

```diff
@@ -105,6 +105,7 @@ def test_sensor_orientation(self):
         self.driver.ephemeris_time = [297088762.61698407]
         self.driver._props = {}
         orientation = self.driver.sensor_orientation
+        print("{:.16f}, {:.16f}, {:.16f}, {:.16f}".format(*orientation[0]))
```

Comment:
does the numpy assert not give enough detail for this?

Reply:
Revmoved